src/CMakeLists.txt

```diff
@@ -55,6 +55,32 @@ set(SOURCE_FILES
 add_library(EIPScanner SHARED ${SOURCE_FILES})
 add_library(EIPScannerS STATIC ${SOURCE_FILES})
 
+message(${PROJECT_SOURCE_DIR}/src)
+
+target_include_directories(EIPScanner PUBLIC
```

Comment:
We already use include_directories(${CMAKE_SOURCE_DIR}/src) in our root CMakeLists.txt. All of the include paths of the project should be relative

Reply:
> I noticed this fork a while ago where someone also implemented windows support, but they never submitted a pull request, and I never reviewed the code - https://github.com/shrddr/EIPScanner
> 
> I don't know if you referenced it or not, but it might be helpful.

I've looked at this. Changes are almost the same, except I didn't touch whitespace, didn't change c++ version and didn't implemented/changed any new logic. Also, it's important to fix SO_RCVTIMEO usage for windows. Today I've tested it on windows and it's stable.